### ISSUE TITLE
node_modules/@vscode/codicons folder contained in sprotty npm package

### ISSUE THREAD

**Issue by kaisalmen** [eclipse/sprotty#311]
We stumbled upon this by accident in an angular project where monaco-editor is included, too. This lead to errors with `ng serve` (angular development server) and `ng test`, because `codicon.ttf`  was contained in different versions. The reason is sprotty npm packages contains **node_modules/@vscode/codicons**  (0.12.0+, haven't checked older builds). I guess this should not be and should be fixed.

While going through issues here, I found that using `codicon.css` directly leads to problems. That's why I have prepared as solution that transforms `codicon.css` and `codicon.ttf` into one generated `codicon.css` (see referenced PR) which is referenced locally. This moves `@vscode/codicons` to a **devDependency** and removes any problems with dependent packages. Question is if you like my proposed solution. 🙂

**Comment by BjBe82** [comment]
Just to add this here since we have now also an issue with this dependency (@vscode/codicons) when upgrading from Angular 13 to Angular 14.

The build of the Angular Application will fail with the following Error:
```
./node_modules/@vscode/codicons/dist/codicon.css:6:0 - Error: Module parse failed: Unexpected character '@' (6:0)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
|  *--------------------------------------------------------------------------------------------*/
| 
> @font-face {
|       font-family: "codicon";
|       src: url("./codicon.ttf?7cea6a135a6c0359d1f6d18e6b2c5c42") format("truetype");

error Command failed with exit code 1.
```
Background: https://github.com/angular/angular-cli/issues/23273

This could be solved by Custom webpack builder for Angular as outlined here: https://bytemeta.vip/repo/microsoft/monaco-editor/issues/3272.  But moving it from a dependency to the devDependencies would be appreciated.

**Comment by kaisalmen** [comment]
This has been resolved with #312
